packages/ui/src/components/ExitIntentModal.tsx

```
@@ -1,145 +1,188 @@
 "use client";
 
-import { useEffect, useState, useRef } from "react";
+import { FC, useEffect, useRef, useState } from "react";
 import type { MouseEvent as ReactMouseEvent } from "react";
```

ESLint

[error] 'ReactMouseEvent' is defined but never used.

(@typescript-eslint/no-unused-vars)